hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/S3GatewayConfigKeys.java

```diff
@@ -64,6 +64,10 @@
   public static final String OZONE_S3G_KERBEROS_PRINCIPAL_KEY =
       "ozone.s3g.kerberos.principal";
 
+  public static final String OZONE_S3G_STREAMING_PUT_ENABLE =
+      "ozone.s3g.streaming.put.enable";
+  public static final boolean OZONE_S3G_STREAMING_PUT_ENABLE_DEFAULT =
+      false;
```

Comment:
Please reuse OzoneConfigKeys.DFS_CONTAINER_RATIS_DATASTREAM_ENABLE instead of adding a new conf.